react/features/base/conference/actions.any.ts

```diff
@@ -847,6 +848,22 @@ export function setFollowMe(enabled: boolean) {
     };
 }
 
+/**
+ * Enables or disables the Follow Me feature used only for the recorder.
+ *
+ * @param {boolean} enabled - Whether Follow Me should be enabled and used only by the recorder.
+ * @returns {{
+ *     type: SET_FOLLOW_ME_RECORDER,
+ *     enabled: boolean
+ * }}
+ */
+export function setFollowMeRecorder(enabled: boolean) {
```

Comment:
I'm wondering why didn't we put this into the follow-me feature?

Reply:
I can refactor it, but the state is now in the conference feature and I kept the new values there.